R/get_shortnames.R

```diff
@@ -0,0 +1,12 @@
+#' get_shortnames
+#' return the shortnames of taxonomy labels
+#'
+#' @param longnames vector; vector of long taxonomic labels
+#' @export
+#' @return vector
+#' @examples
+#' get_shortnames(rownames(abundance_matrix))
```

Comment:
example is not functional. could use package data, or include dummy data in the example